### DISCUSSION TITLE
2005/05/02/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#16] [Announcements]
# 2005/05/02/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-8

https://blog.qizhen.xyz/2005/05/02/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-8

**Comment by ruanqizhen** [comment]
华琳
2005年05月3日 02:25 编辑
不好意思应该是爱国人士！呵呵！


华琳
2005年05月3日 02:24 编辑
这是机器自己弄的数字还是你自己写的啊？？？有意思。谢谢你顶我的文章。谢谢爱过人士的你！
